vizro-core/src/vizro/static/css/layout.css

```diff
@@ -105,7 +116,7 @@
   width: 100%;
 }
 
-.nav-bar {
+#nav-bar {
   align-items: center;
   background-color: var(--surfaces-bg-02);
```

Comment:
It's not change made in this PR, but maybe we should delete `background-color: var(--surfaces-bg-02);` from #nav-bar and icon-button classes, and let those components inherit background color from the #left-sidebar. 

Apart from being unnecessary property, it makes it hard for users to overwrite it, as currently if users want to overwrite the background color of entire left sidebar, they have to overwrite all three classes.

Reply:
That's a great suggestion - thanks! ⭐ 

I've decided to get rid of the `background-color` from all the inner containers and just add it to the outermost container. The inner ones will automatically inherit. Implemented here: https://github.com/mckinsey/vizro/pull/205/commits/f2fb529489c8e6bab78494e72492c319eda39c7d